### ISSUE TITLE
Ammeter button is missing its accessibleName

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#1270]
From #1265 

it is missing the a11y view text:

<img width="930" height="238" alt="Image" src="https://github.com/user-attachments/assets/1d7b2883-01cc-4360-8c10-428a20f1fe84" />
